scripts/track_upstream_commits/track_upstream_commits.py

```diff
@@ -51,30 +51,59 @@ def is_following_branch():
     # Change working directory to the cloned repo
     os.chdir(target_folder)
 
+def convert_iota_to_sui_crates(crates):
```

Comment:
I know this is a bit of nitpicking, and sorry for that... but why do we even talk about crates in the code at all?
Shouldn't the script be valid for all folders in the repository that belong to different codeowners?

I realized that in this function, where we append the `crates/`.

Maybe instead of passing owned crates via parameter, we should pass owned folders with relative paths to the root. And rename all occurenced of crates in the code 😅

Reply:
Or even pass valid regexes like in the codeowners file 🏃

Reply:
You're right, we should track all folders of the codeowner, not just `/crates`.